static/assets/tools.json

```diff
@@ -4854,5 +4854,30 @@
 	"fmuImport": [
               "CS"
 	]
-     }
+    },
+    {
+    "name": "esmini",
+    "license": "osi",
```

Comment:
Shouldn't this be MPL-2,0?

Reply:
In the [guide](https://github.com/modelica/fmi-standard.org/blob/main/CONTRIBUTING.md#updating-the-tools-list) I see only two options: "commercial", "osi"

So I guess at this level it's just a distinction between Commercial and Open Source.

Reply:
Oh I was thinking of Open Simulation Interface (OSI) :)
That's why I was a bit confused.

Reply:
That was my initial confusion as well, but now I learned a new interpretation of the word OSI :)